src/hawtio/plugins/shared/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './components'
+export * from './mbean'
```

Comment:
I'd prefer to name this folder as `tree` as the name is used in the legacy hawtio-integration as well.